CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.2+7
+
+* Strong mode fixes.
```

Comment:
did these come in previous commits?

Reply:
Yes, see https://github.com/dart-lang/initialize/pull/58